cluster-diagnosis/sysdumpcollector.py

```diff
@@ -200,6 +202,27 @@ def collect_cilium_configmap(self):
             log.info("collected cilium configmap yaml file: {}".format(
                 configmap_file_name))
 
+    def collect_cilium_secret(self):
+        secret_file_name = "cilium-etcd-secrets-{}.json".format(
+            utils.get_current_time())
+        cmd = "kubectl get secret cilium-etcd-secrets -n kube-system -ojson"
```

Comment:
for consistency with other code... `-o json`

Reply:
Done